flax/linen/module.py

```diff
@@ -1559,7 +1559,7 @@ def loss(params, perturbations, inputs, targets):
                                   #      [-1.456924   -0.44332537  0.02422847]]
 
     """
-    value += self.variable(collection, name, lambda: jnp.zeros_like(value)).value
+    value += self.variable(collection, name, lambda: jnp.zeros_like(value)).value # type: ignore
```

Comment:
are we messing up internal type annotations in flax module methods to require this opt-out?